SalesforceSDK/Core/Settings/IApplicationInformationService.cs

```diff
@@ -33,7 +33,7 @@ public interface IApplicationInformationService
     {
         Task<string> GetApplicationDisplayNameAsync();
 
-        Task<string> GenerateUserAgentHeaderAsync(bool isHybrid);
+        Task<string> GenerateUserAgentHeaderAsync(bool isHybrid, bool isSmartSync);
```

Comment:
It might be better to add a generic string parameter here that identifies the library. In the future if we add more than ```SmartSync```, this will still work.